packages/react-devtools-shared/src/isArray.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+declare function isArray(a: mixed): boolean %checks(Array.isArray(a));
+
+const isArrayImpl = Array.isArray;
+
+// eslint-disable-next-line no-redeclare
+function isArray(a: mixed): boolean {
+  return isArrayImpl(a);
+}
+
+export default isArray;
```

Comment:
Oops, sorry. The main reason for suggesting this here: https://github.com/facebook/react/pull/22438#issuecomment-927843218

Was that DevTools doesn't use Closure compiler, so the wrapper function _doesn't get inlined_. For this reason, I was suggesting we should just export/use `isArrayImpl` directly (like we do with `hasOwnProperty`.

Reply:
I got it

Reply:
Done